zulipterminal/ui_tools/boxes.py

```diff
@@ -201,16 +201,14 @@ def private_box_view(
             ]
             recipient_info = ", ".join(
                 [
-                    f"{self.model.user_dict[email]['full_name']} <{email}>"
+                    f"""{STATE_ICON[self.model.user_dict.get(email, None).get("status","inactive") if self.model.user_dict.get(email,None) else "inactive"]} {self.model.user_dict[email]['full_name']} <{email}>"""
                     for email in self.recipient_emails
                 ]
             )
         else:
             self._set_regular_and_typing_recipient_user_ids(None)
             self.recipient_emails = []
             recipient_info = ""
-
-        self.send_next_typing_update = datetime.now()
```

Comment:
Unrelated?

Reply:
Unrelated. Forgot to add a commit to fix the errors that sprung up.